docker/dockerfile-install-centos

```diff
@@ -0,0 +1,20 @@
+# Parameters related to building hipcub
+ARG base_image
+
+FROM ${base_image}
+LABEL maintainer="saad.rahim@amd.com"
```

Comment:
We should no longer use any personal email addresses. Please change to hipcub-maintainer@amd.com